content/blog/2018/01/2018-01-06-jep-200.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "JEP-200: Remoting / XStream whitelist integrated into Jenkins core"
+tags:
+- core
+- security
+- remoting
+- upgrade
+author: jglick
+---
+
+== Overview
+
+link:https://github.com/jenkinsci/jep/blob/master/jep/200/README.adoc[JEP-200] has been integrated into Jenkins weekly builds
+and (if all goes well) will be a part of the next LTS line.
+In a nutshell, this change is a security hardening measure
+to be less permissive about deserializing Java classes defined in the Java Platform or libraries bundled with Jenkins.
+For several years now, Jenkins has specifically _blacklisted_ certain classes and packages according to known or suspected exploits;
+now it will reject all classes not explicitly mentioned in a _whitelist_, or defined in Jenkins core or plugins.
+
```

Comment:
Add a "Before Upgrade" section, which explicitly recommends doing a backup before the upgrade (a part of JEP)

Reply:
I would also say that "This upgrade implies a significant risk of regressions. Make sure you backup your data before upgrading. If you have a way to test a change before rolling it to production, we recommend doing so"